### PR TITLE
Time picker

### PR DESCRIPTION
**Pull request by vlandham**
Starting with Airbnb picker.

### CODE REVIEW

**Comment by vlandham** [comment]
Ok. i believe this is working now for time periods where we have data coming in. 

time_in3

Added `options` to actions and api calls to hold `startDate` and `endDate`. These are maintained as moment objects except for where they are serialized / deserialized in the url. Not sure if we want to switch that to using moment dates as well - and stay in moment land the entire time. 

ping @pbeshai

**Comment by pbeshai** [comment]
I like the idea of being in moment land the entire time. Just update the encodeDate/decodeDate functions to do it

File: src/api/api.js [hunk 1]
```diff
@@ -27,10 +49,13 @@ export function getLocationInfo(locationId) {
...
  * @param {String} locationId The location to query (e.g., nauswaseattle)
+ * @param {Options} object Options with startDate and endDate
  * @return {Promise} A promise after the get request was made
  */
-export function getLocationTimeSeries(timeAggregation, locationId) {
-  return get(`/locations/${locationId}/time/${timeAggregation}/metrics`)
+export function getLocationTimeSeries(timeAggregation, locationId, options = {}) {
+  const params = getDateRangeParams(timeAggregation, options);
+  const paramString = params.length > 0 ? `?${params.join('&')}` : '';
+  return get(`/locations/${locationId}/time/${timeAggregation}/metrics${paramString}`)
```
Comment (pbeshai) [inline src/api/api.js]: You don't have to do this paramString stuff. `get` takes the query parameters as its second arguments. Just pass it in as an object.

File: src/assets/datepicker.css [hunk 2]
```diff
@@ -0,0 +1,512 @@
...
+  .SingleDatePicker__close svg {
+    height: 15px;
+    width: 15px;
+    fill: #cacccd; }
+  .SingleDatePicker__close:hover, .SingleDatePicker__close:focus {
+    color: #b0b3b4;
+    text-decoration: none; }
+
+.SingleDatePickerInput {
+  border: 1px solid #dbdbdb; }
```
Comment (pbeshai) [inline src/assets/datepicker.css]: Something tells me we can reduce the total amount of this we need if we are only using the date range picker.

**Comment by pbeshai** [comment]
Looks good Jim +1 nice work

**Comment by vlandham** [comment]
thanks for the feedback.
fixed params use and now date serialization uses moment.

**Pull request merged**